src/integration-test/java/com/commercetools/sync/integration/commons/utils/CustomerITUtils.java

```diff
@@ -19,6 +57,118 @@ public static void deleteCustomers(@Nonnull final SphereClient ctpClient) {
         queryAndExecute(ctpClient, CustomerQuery.of(), CustomerDeleteCommand::of);
     }
 
+    public static ImmutablePair<Customer, CustomerDraft> createSampleCustomerJohnDoe(
+        @Nonnull final SphereClient ctpClient) {
+
+        final Store storeBerlin = createStore(ctpClient, "store-berlin");
+        final Store storeHamburg = createStore(ctpClient, "store-hamburg");
+        final Store storeMunich = createStore(ctpClient, "store-munich");
+
+        final Type customTypeGoldMember = createCustomerCustomType("customer-type-gold", Locale.ENGLISH,
+            "gold customers", ctpClient);
+
+        final CustomerGroup customerGroupGoldMembers = createCustomerGroup(ctpClient, "gold members", "gold");
```

Comment:
This 5 resources could be created in parallel to speed up the whole testing.

Reply:
I was aware of this but then forgot about it, good catch :)